NEWS.md

```diff
@@ -136,6 +136,9 @@ Language changes
     backslashes and the end of the literal while 2n+1 backslashes followed by a quote encodes n
     backslashes followed by a quote character ([#22926]).
 
+  * `readdir()` output is now guaranteed to be sorted on any OS. The keyword `sort` allows to
+    optionally turn off explicit sorting ([#24626]).
```

Comment:
"allows to" is not grammatical in this context … you need to add a subject, like "allows one to…".  (Apparently this is a common mistranslation from *ermöglicht* in German?)

A more natural phrasing might be: "A new `sort` keyword argument can be used to disable explicit sorting."

Reply:
I had *erlaubt es* in mind. I guess impersonal, complicated formulations are natural for Germans :)